keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/AuthorizationSupport.kt

```diff
@@ -103,15 +104,16 @@ class AuthorizationSupport(
         DELIVERY_CONFIG -> repository.getDeliveryConfig(identifier).application
         else -> throw InvalidRequestException("Invalid target type ${target.name} for application permission check")
       }
-      permissionEvaluator.hasPermission(auth, application, "APPLICATION", action.name)
-        .also { allowed ->
-          log.debug("[ACCESS {}] User {}: {} access to application {}.",
-            allowed.toAuthorization(), auth.principal, action.name, application)
-
-          if (!allowed) {
-            throw AccessDeniedException("User ${auth.principal} does not have access to application $application")
-          }
+      AuthenticatedRequest.allowAnonymous {
```

Comment:
Calls to Fiat are made anonymous because we don't have access to the request headers in the library.